src/main/java/bio/terra/cli/service/ServerManager.java

```diff
@@ -74,7 +74,7 @@ public boolean pingServerStatus() {
   }
 
   // This variable defines the server that the CLI points to by default.
-  private static final String DEFAULT_SERVER_FILENAME = "terra-dev.json";
+  private static final String DEFAULT_SERVER_FILENAME = "wchamber-dev.json";
```

Comment:
Aha – I'm guessing this is intended to (1) not require being on the VPN, and also (2) provide a more stable version of WSM to prevent breakages due to just-merged PRs or whatnot. (Depends somewhat on Will not doing too much with his personal WSM, which I guess is safe at least for awhile?)

Nice approach!

Reply:
The intent was just (1) -- so that Willy and Nicole don't need to be on the VPN to test the CLI. I'd prefer to use WSM dev long-term so that we don't have to worry about maintaining a separate instance.

Credit to @wchamber for the workaround here -- I didn't know personal dev environments weren't behind the firewall!

Reply:
Yep, 2) is actually a downside. I don't expect my personal environment to be more stable and will need to ad-hoc fix or monitor it.